CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.7.4] - 2024-06-06
```

Comment:
We will keep this PR open until all tasks planned for this milestone finished and then update this date.